.gitignore

```diff
@@ -0,0 +1 @@
+.vscode
```

Comment:
why add this?

Reply:
VS Code is a popular editor.  I was surprised that there was not a gitignore for this.